recipes/hmftools-linx/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.4" %}
+{% set sha256 = "0036480db0048a5cf54032421791a372b94add2743b271fdcaf77ff053850efe" %}
+
+package:
+  name: hmftools-linx
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/hartwigmedical/hmftools/releases/download/sv-linx-v{{ version }}/sv-linx_{{ version }}.jar
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - zlib
+    - openjdk >=8
+    - xorg-libxtst
+    - r-base ==3.6.*
+    - r-tidyr
+    - r-dplyr
+    - r-ggplot2
+    - r-cowplot
+    - r-magick
+    - bioconductor-gviz
+    - circos>=0.69.6                   # for circos
+    - libgd                            # for circos
```

Comment:
cirocos should bring this in on there own, you don't need to specify it here I think